.github/workflows/pull-request.yml

```diff
@@ -19,44 +19,28 @@ env:
     CODECOV_UNIQUE_NAME: CODECOV_UNIQUE_NAME-${{ github.run_id }}-${{ github.run_number }}
           
 jobs:
-  Linter:
-    name: Check for linting errors
+  Code-Quality-Checks:
+    name: Check for linting, formatting and typpe errors
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

Comment:
Create a `-name` entry here to make sure we have an easy to follow event trail in the logs.

Reply:
We did this in the other repo.

Reply:
@palisadoes 
Line 23 has the name entry for the particular job. And then each of the steps have a name attribute as well.
This is analogous to Talawa-Admin repo.
![image](https://user-images.githubusercontent.com/96648934/230040909-30dfd8c0-bcd4-441a-838f-5b1ec6a866b2.png)